### PR TITLE
Adds Lists

### PR DESCRIPTION
**Pull request by samuelechu**
image

### CODE REVIEW

**Comment by djblue** [comment]
Do we need to show the query status for lists?

**Comment by samuelechu** [comment]
> Do we need to show the query status for lists?

DDF shows a status for lists, we could remove it if it's unnecessary though.
image

File: src/main/webapp/intrigue-api/attributes.json [hunk 1]
```diff
@@ -417,7 +417,7 @@
   {
     "id": "lists",
     "multivalued": true,
-    "type": "XML"
+    "type": "JSON"
```
Comment (djblue) [inline src/main/webapp/intrigue-api/attributes.json]:
```suggestion
    "type": "MetacardAttributes"
```

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     "typescript": "^3.6.4"
   },
   "dependencies": {
-    "@apollo/react-hooks": "^3.1.2",
+    "@apollo/react-hooks": "^3.1.3",
```
Comment (samuelechu) [inline package.json]: https://github.com/apollographql/react-apollo/issues/3690

Comment (samuelechu) [reply package.json]: https://github.com/connexta/revelio/pull/115/files#diff-33686393fc5dffe7c4e217ac20f689d0R58

**Pull request merged**